lib/hackerone/client.rb

```diff
@@ -225,7 +125,12 @@ def self.parse_response(response)
         elsif response.status.to_s.start_with?("5")
           raise RuntimeError, "API called failed, probably their fault: #{response.body}"
         elsif response.success?
-          JSON.parse(response.body, :symbolize_names => true)[:data]
+          response_body_json = JSON.parse(response.body, :symbolize_names => true)
```

Comment:
Interesting. Did this fix a bug?

Reply:
I think so; I didn't verify in the old code but when changing a reference the response doesn't contain `data` as a top-level key. You can see that here:

```
@report = response_json[:relationships][:report][:data]
```